travis/check-stylesheet.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# This file is part of the phpBB Forum Software package.
+#
+# @copyright (c) phpBB Limited <https://www.phpbb.com>
+# @license GNU General Public License, version 2 (GPL-2.0)
+#
+# For full copyright and license information, please see
+# the docs/CREDITS.txt file.
+#
+set -e
+set -x
```

Comment:
removes it please, or add set +x just before sourcing nvm because the output is 1000 times too verbose